### PR TITLE
Added ETHSamba Hack Event

### PR DESCRIPTION
**Pull request by xdaluca**
## Description

Added the ETHSamba Hack community event happening at the end of March 2023

## Related Issue

No related issues.

### CODE REVIEW

File: src/data/community-events.json [hunk 1]
```diff
@@ -710,4 +710,13 @@
     "startDate": "2022-11-28",
     "endDate": "2022-12-4"
   }
+    {
```
Comment (corwintines) [inline src/data/community-events.json:713]:
```suggestion
  {
```

File: src/data/community-events.json [hunk 2]
```diff
@@ -710,4 +710,13 @@
     "startDate": "2022-11-28",
     "endDate": "2022-12-4"
   }
```
Comment (corwintines) [inline src/data/community-events.json:712]:
```suggestion
  },
```

**Review by corwintines — CHANGES_REQUESTED** [review]

**Comment by xdaluca** [comment]
Hi @corwintines committed the changes thanks!

**Review by corwintines — APPROVED** [review]
Thanks @xdaluca

**Comment by xdaluca** [comment]
@corwintines how long does it usually take for it to be on the live site?

**Pull request merged**